ign-gazebo4.yaml

```diff
@@ -50,8 +50,8 @@ repositories:
   ign-transport:
     type: git
     url: https://github.com/ignitionrobotics/ign-transport
-    version: ign-transport8
+    version: master
   sdformat:
     type: git
     url: https://github.com/osrf/sdformat
-    version: sdf9
+    version: master
```

Comment:
SDF development is a bit different from the rest now. We have the `sdf10` branch for Dome and `master` will be SDF 11, for Ignition-E